src/openstack_manager.py

```diff
@@ -2,58 +2,45 @@
 # See LICENSE file for licensing details.
 
 """Module for handling interactions with OpenStack."""
+import json
 import logging
-from pathlib import Path
+import secrets
+from dataclasses import dataclass
 
+# subprocess module is used to call image build script.
+from subprocess import SubprocessError  # nosec
```

Comment:
I know it may be confusing, but its another SubprocessError that you need to catch, its from `errors.SubprocessError`.

Reply:
You're right, thanks!